charts/datadog/values.yaml

```diff
@@ -339,6 +339,11 @@ datadog:
   # datadog.criSocketPath -- Path to the container runtime socket (if different from Docker)
   criSocketPath:  # /var/run/containerd/containerd.sock
 
+  # Configure how the agent interact with the host's container runtime
+  containerRuntimeSupport:
+    # datadog.containerRuntimeSupport.enabled -- Set this to false to disable agent access to container runtime.
+    enabled: true
```

Comment:
I’m wondering if, in order to ease use GKE autopilot, we couldn’t have an `auto` value and a helper function that would return `true` if `containerRuntimeSupport.enabled or not providers.gke.autopilot`.